packages/sui-studio/bin/migrations/test-folders.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-console */
+
+const glob = require('fast-glob')
+const fs = require('fs-extra')
+
+function migrateTestFolders() {
+  const files = glob.sync(['test/*/*/index.js'])
+
+  files.forEach(file => {
+    const [category, component] = file
+      .replace('/index.js', '')
+      .replace('test/', '')
+      .split('/')
+
+    const testPath = `./test/${category}/${component}`
+    const newTestPath = `./components/${category}/${component}/test`
+
+    if (!fs.existsSync(newTestPath)) {
+      fs.moveSync(testPath, newTestPath)
+      console.log(`Moved folder: ${testPath}`)
+      fs.moveSync(`${testPath}/index.js`, `${newTestPath}/index.test.js`)
```

Comment:
Given it is already moved 👇 
```suggestion
      fs.moveSync(`${newTestPath}/index.js`, `${newTestPath}/index.test.js`)
```

Reply:
Yep, thanks!